src/tools/annotation/LengthTool.js

```diff
@@ -137,27 +136,25 @@ export default class LengthTool extends BaseAnnotationTool {
     // Calculate the length, and create the text variable with the millimeters or pixels suffix
     const length = Math.sqrt(dx * dx + dy * dy);
 
-    // Pixel diagonal
-    const uncertainty = colPixelSpacing
-      ? decimal(
-          colPixelSpacing * colPixelSpacing + rowPixelSpacing * rowPixelSpacing
-        ).sqrt()
-      : decimal.sqrt(2);
-
-    // Store the length inside the tool for outside access
-    const [roundedLength, roundedDigits] = rounding.roundValue(
-      length,
-      uncertainty
-    );
-    const [roundedUncertainty, roundedUncertaintyDigits] = rounding.roundValue(
-      uncertainty,
-      uncertainty
+    const uncertainty = measurementUncertainty.getPixelDiagonal(
+      colPixelSpacing,
+      rowPixelSpacing
     );
 
+    const roundedLength = colPixelSpacing
+      ? measurementUncertainty.roundValueBasedOnUncertainty(length, uncertainty)
+      : roundToDecimal(length, 1);
+
+    const roundedUncertainty = colPixelSpacing
+      ? measurementUncertainty.roundValueBasedOnUncertainty(
+          uncertainty,
+          uncertainty
+        )
+      : roundToDecimal(uncertainty, 1);
+
     data.invalidated = false;
-    data.uncertainty = roundedUncertainty;
     data.length = roundedLength;
-    data.digits = roundedDigits;
```

Comment:
why was this removed?

Reply:
digits vary depending on the uncertainty rounding and we dont need cornerstone to limit the number of digits. The data.digits was already overwritten by Kathi's german temp localisation module earlier